src/api/volume-snapshots.tsx

```diff
@@ -0,0 +1,159 @@
+import {
+  continueOrFinish,
+  handleResponse,
+  pushFailure,
+  pushSuccess,
+} from "util/helpers";
+import { LxdOperationResponse } from "types/operation";
+import { LxdStorageVolume, LxdVolumeSnapshot } from "types/storage";
+import { LxdApiResponse, LxdSyncResponse } from "types/apiResponse";
+import { EventQueue } from "context/eventQueue";
+
+export const createSnapshot = (args: {
```

Comment:
I wonder if the function names should be unique. We are now introducing two functions with the same name `createSnapshot` (and similar for the others in this file below). Might be too easy to mix them up. Better keep them specific, like `createVolumeSnapshot`?